chocolatey/img
# (add) Event images for Coding Livestream

## Description Of Changes
These images will appear on the event page for the Coding Livestream.

## Motivation and Context
We need new images for the event.

## Testing
1. Run the website
2. Open the right side flyout and notice there is now and image for the "Coding Livestream" event. 

## Change Types Made

## Change Checklist


## Related Issue
* https://github.com/chocolatey/choco-theme/issues/380
* https://github.com/chocolatey/chocolatey.org/issues/303
* https://github.com/chocolatey/home/issues/297
* https://gitlab.com/chocolatey/community-infrastructure/community.chocolatey.org/-/issues/1246
